To: vim_dev@googlegroups.com
Subject: Patch 7.3.387
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.387 (after 7.3.386)
Problem:    Test 83 may fail for some encodings.
Solution:   Set 'encoding' to utf-8 earlier.
Files:	    src/testdir/test83.in


*** ../vim-7.3.386/src/testdir/test83.in	2011-12-23 14:56:24.000000000 +0100
--- src/testdir/test83.in	2011-12-30 13:05:05.000000000 +0100
***************
*** 2,13 ****
  
  STARTTEST
  :so mbyte.vim
  :if !has('iconv') || iconv("\x82\x60", "cp932", "utf-8") != "\uff21"
  : e! test.ok
  : w! test.out
  : qa!
  :endif
- :set enc=utf8
  
  :/^text for tags1$/,/^text for tags1$/+1w! Xtags1.txt
  :/^text for tags2$/,/^text for tags2$/+1w! Xtags2.txt
--- 2,13 ----
  
  STARTTEST
  :so mbyte.vim
+ :set enc=utf8
  :if !has('iconv') || iconv("\x82\x60", "cp932", "utf-8") != "\uff21"
  : e! test.ok
  : w! test.out
  : qa!
  :endif
  
  :/^text for tags1$/,/^text for tags1$/+1w! Xtags1.txt
  :/^text for tags2$/,/^text for tags2$/+1w! Xtags2.txt
*** ../vim-7.3.386/src/version.c	2011-12-23 14:56:24.000000000 +0100
--- src/version.c	2011-12-30 13:06:17.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     387,
  /**/

-- 
Mental Floss prevents moral decay!

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
